src/core/Akka.MultiNodeTestRunner/Akka.MultiNodeTestRunner.nuspec

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2013/05/nuspec.xsd">
+  <metadata>
+    <id>Akka.MultiNodeTestRunner</id>
+    <version>1.4.23</version>
+    <title>Akka.MultiNodeTestRunner</title>
+    <authors>Akka.NET Team</authors>
+    <owners>Akka.NET Team</owners>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <licenseUrl>https://github.com/akkadotnet/akka.net/blob/master/LICENSE</licenseUrl>
+    <projectUrl>https://github.com/akkadotnet/akka.net</projectUrl>
+    <iconUrl>http://getakka.net/images/AkkaNetLogo.Normal.png</iconUrl>
+    <description>Akka.NET Multi-node Test Runner; used for executing tests written with Akka.Remote.TestKit</description>
+    <copyright>Copyright � 2013-2021 .NET Foundation</copyright>
+    <tags>akka actors actor model Akka concurrency</tags>
+  </metadata>
+  <files>
```

Comment:
In order to publish a Linux version of our NuGet packages.... Maybe we should think about doing a .NET global tool